hieradata/role/fiberspec.yaml

```diff
@@ -2,3 +2,4 @@
 classes:
```

Comment:
It appears that I messed when reviewing https://github.com/lsst-it/lsst-control/pull/1134 as there is not a spec file covering this role (there is an auto-magic catchall role test for that cases but it isn't really enough coverage).

Please add a `spec/hosts/roles/fiberspec_spec.yaml`.  Within that file, please include the `dco` shared example group. E.g. `include_examples 'dco'`